Security/Data_Classification.mediawiki

```diff
@@ -86,7 +103,6 @@ This information, if disclosed beyond the individuals, would have a significant
 
 * Information shared in the monthly MoCo/MoFo internal meeting.
 * Bugzilla bugs with the "Moco confidential" or "infrastructure" flags.
-* Mozilla's employe phonebook.
```

Comment:
Shouldn't this be replaced with a mention to the Mozilla People Directory (used by staff and non-staff)?

Reply:
I don't think so. This section is giving examples of data classified as `Mozilla Confidential - Staff and NDA'd Mozillians Only`. Though phonebook was classified as that (since it was only accessibly by staff and only contained staff data), people.mozilla.org is accessibly publicly on the internet by anyone and contains data classified at all different levels, not just `Mozilla Confidential - Staff and NDA'd Mozillians Only`

Reply:
ok!